app/models/observation_view.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# Keep track of when each user last viewed each observation.
+class ObservationView < AbstractModel
+  belongs_to :observation
+  belongs_to :user
+
+  def self.update_view_stats(observation, user)
+    return if observation.blank? || user.blank?
+
+    if view = find_by(observation: observation, user: user)
```

Comment:
To avoid RuboCop complaint:
```suggestion
    if (view = find_by(observation: observation, user: user))
```

Reply:
PS: The cop is [Lint/AssignmentInCondition](https://docs.rubocop.org/rubocop/0.85/cops_lint.html#lintassignmentincondition)
You're not seeing a complaint because .rubocop_todo.yml is squelching the cop because we so many offenses (50).

Reply:
Thanks, didn't know, will do.